### PR TITLE
complete partial functionaties of browsing page and set up some angul…

### PR DESCRIPTION
**Pull request by yzhang54**
…ar config

### CODE REVIEW

**Comment by yzhang54** [comment]
Don't worry about the CSS for now; I will add more style later.

File: src/app/pages/browse/browse.component.ts [hunk 1]
```diff
@@ -1,15 +1,96 @@
...
+    {value: '17', viewValue: 'Li'},
+    {value: '18', viewValue: 'Kazakh'},
+    {value: '19', viewValue: 'Dai'},
+    {value: '20', viewValue: 'She'},
+    {value: '21', viewValue: 'Lisu'},
+    {value: '22', viewValue: 'Dongxiang'},
+    {value: '23', viewValue: 'Gelao'},
+    {value: '24', viewValue: 'Lahu'},
+    {value: '25', viewValue: 'Wa'},
+    {value: '26', viewValue: 'Sui'},
```
Comment (JOOOEG123) [inline src/app/pages/browse/browse.component.ts:50]: Nice, this looks. We would use this.

File: src/styles.scss [hunk 2]
```diff
@@ -1,4 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
+@import "~@angular/material/prebuilt-themes/indigo-pink.css";
+
```
Comment (JOOOEG123) [inline src/styles.scss:3]: we use bootstrap and not material ui

File: tsconfig.app.json [hunk 3]
```diff
@@ -3,7 +3,8 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "./out-tsc/app",
-    "types": []
+    "types": [],
+    "paths": { "@angular/*": [ "../node_modules/@angular/*" ] }
```
Comment (JOOOEG123) [inline tsconfig.app.json:7]: making changes here could break the application.

File: src/app/pages/browse/browse.component.ts [hunk 4]
```diff
@@ -1,15 +1,96 @@
...
   constructor() { }
+  
 
   ngOnInit(): void {
+    //throw new Error('Method not implemented.');
   }
 
+  filterDropDown() {
+    this.active = !this.active;
+  }
```
Comment (JOOOEG123) [inline src/app/pages/browse/browse.component.ts:94]: This is correct, I will show you possible next step.

**Review by JOOOEG123 — CHANGES_REQUESTED** [review]
You made a lot of good changes. I think you unerstand more about angular now.  Just a reminder we dont use material we use boostrap and ngx bootstrap. It has similar function as material and have all the scss class you will need. Let meet tomorrow. :) Great Job on creating you fire Angular PR.

File: src/app/pages/browse/browse.component.html [hunk 5]
```diff
@@ -1 +1,71 @@
...
+
+        <section class="gender-section">
+          <label class="gender-margin">Gender</label>
+          <mat-radio-group [(ngModel)]="genderPosition">
+            <mat-radio-button class="gender-margin" value="male">male</mat-radio-button>
+            <mat-radio-button class="gender-margin" value="female">female</mat-radio-button>
+          </mat-radio-group>
+        </section>
+
+        <section class="status-section">
```
[the remark this answers is not in the document]
Comment (yzhang54) [reply src/app/pages/browse/browse.component.html:28]: Could you pls clarify the reasons why we switched? And would it be possible we combine both of them?

File: src/app/pages/browse/browse.component.html [hunk 6]
```diff
@@ -1 +1,71 @@
...
+        </mat-form-field>
+      
+      </form>
+
+    </div>
+
+  </div>
+
+</body>
+
```
[the remark this answers is not in the document]
Comment (yzhang54) [reply src/app/pages/browse/browse.component.html:69]: Will fix later, thanks.

**Pull request closed without merging**